dev/format_code.sh

```diff
@@ -34,13 +36,14 @@ if [ "$DOCKER" == "--docker" ]; then
 
   if [ "$(docker images -q $DOCKER_IMAGE | wc -l)" == "0" ]; then
     echo "Building clang-format container..."
-    cd "$REPO_ROOT/dev/clang-format"
-    docker build . -t librepcb/clang-format:6
-    cd -
+    (
+      cd "$REPO_ROOT/dev/clang-format"
+      docker build . -t librepcb/clang-format:6
+    )
```

Comment:
An alternative might be to remove the `cd` and pass the path to `docker build` instead...

Reply:
That would indeed be simpler and cleaner.

Reply:
Fixed.